test/pythonsdk/README.md

```diff
@@ -42,6 +42,22 @@ Then execute the integration tests with command:
 $ tox -e oran-tests
 ```
 
+#### Network Slicing Option2 Preparation script
```

Comment:
Not quite sure why network slicing info is here ? that should not be in the README of the tests